scraper/metadata.yaml

```diff
@@ -0,0 +1,7 @@
+type: service
+github_project: open-telemetry/opentelemetry-collector
+
+status:
+  class: pkg
+  stability:
+    development: [metrics]
```

Comment:
Are traces & logs in scope too?

Reply:
I think we will extend to logs at one point, see request https://github.com/open-telemetry/opentelemetry-collector/issues/11238#issuecomment-2457770329.

For me the first part is to move what we have (only metrics) to this new framework proposed in the issue, then we can extend to logs.